internal/cli/command/cluster/integratedservice/services/restore/common.go

```diff
@@ -0,0 +1,79 @@
+// Copyright © 2020 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package restore
+
+import (
+	"context"
+
+	"emperror.dev/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/banzaicloud/banzai-cli/.gen/pipeline"
+	"github.com/banzaicloud/banzai-cli/internal/cli"
+	"github.com/banzaicloud/banzai-cli/internal/cli/input"
+)
+
+func NewRestoreCommand(banzaiCli cli.Cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "restore",
+		Short: "Restore the cluster from a backup.",
+	}
+
+	cmd.AddCommand(
+		newListCommand(banzaiCli),
+		newResultCommand(banzaiCli),
+		newDeleteCommand(banzaiCli),
+		newRestoreCreateCommand(banzaiCli),
```

Comment:
**Question**: why is this function infixed with `Restore`? I couldn't find any clashing names in the package and it felt inconsistent compared to the rest.

Reply:
You're right, i changed the name of the function